torch/onnx/_internal/fx/_pass.py

```diff
@@ -214,21 +218,26 @@ def run(self, *args, **kwargs) -> torch.fx.GraphModule:
         )
         # Gather graph information before transform.
         old_readable_graph = self.module.print_readable(print_output=False)
-        old_tabular = fx_graph_tabular(self.module.graph)
+        old_tabular = maybe_fx_graph_tabular(self.module.graph)
```

Comment:
I think `fx_graph_tabular` returning None is clear and ok. For example, `dict.get()` doesn't have to be `dict.maybe_get()`.

I view a `maybe` function with the meaning of "do this, but if you can't do nothing and don't fail." Bottomline I would add it when it increases clarity. Here I think the original name w/ the type annotation is clear.

Reply:
I'm not certain of this. In my mind `fx_graph_tabular` is `str` typed. `None` is not valid output. So `maybe` prefix hints why potentially None can be returned.